contracts/staking/ExampleRewardCalculator.sol

```diff
@@ -0,0 +1,28 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.25;
+
+import {IRewardCalculator} from "./interfaces/IRewardCalculator.sol";
+
+contract ExampleRewardCalculator is IRewardCalculator {
+    uint256 public constant SECONDS_IN_YEAR = 31536000;
+
+    /**
+     * @notice A linear, non-compounding reward calculation that rewards a set percentage of tokens per year.
+     *
+     * @param rewardBasisPoints annual reward percentage in basis points (units of 0.1%).
+     */
+    function calculateReward(
+        uint256 stakeAmount,
+        uint64 startTime,
+        uint64 endTime,
+        uint256, // initialSupply
+        uint256, // endSupply
+        uint256 rewardBasisPoints
+    ) external pure returns (uint256) {
+        return stakeAmount * rewardBasisPoints * (endTime - startTime) / SECONDS_IN_YEAR / 1000;
```

Comment:
```suggestion
        return stakeAmount * (rewardBasisPoints  / 1000 ) * (endTime - startTime) / SECONDS_IN_YEAR;
```
Suggest to associate the `1000` factor with the bp param to improve readability.

Reply:
That would equal 0, unfortunately. This is all integer arithmetic, solidity doesn't have floating point numbers, so all the division is at the end to prevent truncation.

Reply:
Ah, right duh